nservicebus/operations/tuning.md

```diff
@@ -31,6 +31,8 @@ You can define a maximum value for the number of messages per second that the en
 NServiceBus will not enforce any throughput restrictions by default.
 
 
+NOTE: Currently version 6 of NServiceBus does not support `MaximumMessageThroughputPerSecond` setting.
```

Comment:
"Currently"? so it will in the future? what about when someone is reading this in the v7 timeframe?  is there any more info we need to give on top of "does not support"?